UnityProject/Assets/Scripts/ConfigurationSaves/ConfigurationSaves.cs

```diff
@@ -0,0 +1,294 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using Newtonsoft.Json;
+using UnityEngine;
+
+namespace ConfigurationSaves
```

Comment:
not a fan of the name for this namespace and folder. Also shouldn't it go inside ``Core``? 

I think a name like ``Filesystem`` makes more sense.

Reply:
well needs to be a separate assembly for drag-and-drop into the Managed files

Reply:
but update Name

Reply:
> well needs to be a separate assembly for drag-and-drop into the Managed files

uhm, does making the asmdef inside ``Core`` make it awkward to import?

Reply:
where is core?

Reply:
I can't see any core DLL?